lib/hawkular/base_client.rb

```diff
@@ -146,14 +146,8 @@ def token_header
     end
 
     def tenant_header
-      if @options[:tenant].nil?
-        { 'Hawkular-Tenant' => 'hawkular' }
-      elsif @options[:tenant].empty?
-        {}
-      else
-        { :'Hawkular-Tenant' => @options[:tenant],
-          'tenantId'         => @options[:tenant] }
-      end
+      @options[:tenant].nil? ? {} : { :'Hawkular-Tenant' => @options[:tenant],
+                                      'tenantId'         => @options[:tenant] }
```

Comment:
isn't here the same information sent twice (2 keys with same value)? what am I missing?